### PR TITLE
Remove duplicated rules

### PR DESCRIPTION
**Pull request by yizhao1**
Signed-off-by: Yi Zhao <yi.zhao@windriver.com>

### CODE REVIEW

File: policy/modules/system/logging.te [hunk 1]
```diff
@@ -605,8 +605,6 @@ optional_policy(`
 
 optional_policy(`
 	udev_read_db(syslogd_t)
-	# for systemd-journal to read seat data from /run/udev/data
-	udev_read_runtime_files(syslogd_t)
```
Comment (pebenito) [inline policy/modules/system/logging.te:609]: This isn't duplicate see discussion in #230

File: policy/modules/services/xserver.te [hunk 2]
```diff
@@ -883,7 +883,7 @@ optional_policy(`
 ')
 
 optional_policy(`
-	resmgr_stream_connect(xdm_t)
+	resmgr_stream_connect(xserver_t)
```
Comment (pebenito) [inline policy/modules/services/xserver.te:886]: I'm ok with this being in the PR, but it should be a separate commit since it is a correction.

File: policy/modules/system/logging.te (hunk 1 above)
Comment (yizhao1) [reply policy/modules/system/logging.te:609]: If this is only for systemd-journal, there is alaredy such line (line 557) in init_systemd conditon block.

File: policy/modules/services/xserver.te (hunk 2 above)
Comment (pebenito) [reply policy/modules/services/xserver.te:886]: I think this is still relevant.
